allensdk/brain_observatory/behavior/stimulus_processing.py

```diff
@@ -77,6 +98,8 @@ def get_images_dict(pkl):
             meta = dict(
                 image_category=cat.decode("utf-8"),
                 image_name=img_name.decode("utf-8"),
+                phase=np.NaN,
```

Comment:
I thought that Doug wanted to encode the phase and frequency in the name of the grating. Can you please confirm?

Reply:
I emailed Doug and asked for him to confirm and validate the data and I'm waiting to hear back from him.

Reply:
Did you get confirmation on this?

Reply:
Nvm, I see a comment below.